testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -6085,19 +6083,278 @@ fn fuzzed_median_fee_rate_estimation_test() {
 
     // Check that:
     // 1) The cost is always the same.
-    // 2) The estimated fee rate only either i) moves up, ii) stays within random noise of where it
-    //    was.
+    // 2) Only grows.
     for i in 1..response_estimated_costs.len() {
         let curr_rate = response_top_fee_rates[i] as f64;
         let last_rate = response_top_fee_rates[i - 1] as f64;
-        // Check that either 1) curr_rate is bigger, or 2) is only smaller by random 11% noise.
-        assert!(curr_rate - last_rate > -1.0 * curr_rate * 0.11);
+        // Check that the rate is growing monotonically.
+        assert!(curr_rate >= last_rate);
```

Comment:
need to investigate whether this is going to be flaky.. might need to wait a few extra rounds to be sure that the tx's have all been incorporated

Reply:
fixed